admin_scripts/ubuntu/ubuntu_16.04/xscreensaver/xscreensaver_enable.sh

```diff
@@ -3,6 +3,7 @@
 # Disable Gnome screensaver, install A LOT of graphic screensavers,
 # enable screensaver for user by default.
 
+export DISPLAY=:0.0
```

Comment:
Er... why? Quite apart from the fact that running things in the user's context like this is not a) ideal; or b) supported, this script doesn't even _try_ to do that!